jenkins.yml

```diff
@@ -52,6 +59,11 @@
         password: "{{ jenkins_admin_password |password_hash('sha512')}}"
         shell: /bin/bash
 
+    - name: Ensure libselinux-python is installed
```

Comment:
Could you please change "libselinux-python" to "libsemanage-python" for consistency?